Examples/ByFeature/TaggedExamples/WebTesting.feature

```diff
@@ -0,0 +1,27 @@
+﻿Feature: Web login testing
+
+    Scenario Outline: Successful login
+    Given I use browser <browser>
+    When I try to login using username <username> and password <password>
+    Then I am logged in
+
+    Examples:
+    | browser       |
+    | Firefox       |
+    | Edge      |
```

Comment:
put | at the same positions like lines above and below

Reply:
Thanks, done.